### PR TITLE
Mood

### PR DESCRIPTION
**Pull request by LarryHsiao**

### CODE REVIEW

File: app/src/main/java/com/larryhsiao/nyx/android/jot/JotContentFragment.java [hunk 1]
```diff
@@ -10,17 +10,22 @@
 import android.text.Editable;
 import android.text.InputType;
 import android.text.TextWatcher;
+import android.util.TypedValue;
```
Comment (LarryHsiao) [inline app/src/main/java/com/larryhsiao/nyx/android/jot/JotContentFragment.java]: Codacy Issue found: [Unused import - android.util.TypedValue.](https://app.codacy.com/manual/LarryHsiao/Nyx/pullRequest?prid=5095791)

**Comment by LarryHsiao** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
- Added 4
           

Complexity increasing per file
==============================
- app/src/main/java/com/larryhsiao/nyx/jots/ConstJot.java  1
- app/src/main/java/com/larryhsiao/nyx/android/jot/JotContentFragment.java  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/LarryHsiao/Nyx/pullRequest?prid=5095791&bid=16761474)

File: app/src/main/java/com/larryhsiao/nyx/JotApplication.java [hunk 2]
```diff
@@ -8,6 +8,9 @@
 import com.silverhetch.clotho.database.SingleConn;
 import com.silverhetch.clotho.database.h2.EmbedH2Conn;
 import com.silverhetch.clotho.source.ConstSource;
+import org.flywaydb.core.Flyway;
+import org.flywaydb.core.api.MigrationVersion;
```
Comment (LarryHsiao) [inline app/src/main/java/com/larryhsiao/nyx/JotApplication.java]: Codacy Issue found: [Unused import - org.flywaydb.core.api.MigrationVersion.](https://app.codacy.com/manual/LarryHsiao/Nyx/pullRequest?prid=5095791)

Comment (LarryHsiao) [inline app/src/main/java/com/larryhsiao/nyx/JotApplication.java]: Codacy Issue found: [Avoid unused imports such as 'org.flywaydb.core.api.MigrationVersion'](https://app.codacy.com/manual/LarryHsiao/Nyx/pullRequest?prid=5095791)

File: app/src/main/java/com/larryhsiao/nyx/android/jot/JotContentFragment.java (hunk 1 above)
Comment (LarryHsiao) [inline app/src/main/java/com/larryhsiao/nyx/android/jot/JotContentFragment.java]: Codacy Issue found: [Avoid unused imports such as 'android.util.TypedValue'](https://app.codacy.com/manual/LarryHsiao/Nyx/pullRequest?prid=5095791)

**Comment by LarryHsiao** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
           

Complexity increasing per file
==============================
- app/src/main/java/com/larryhsiao/nyx/jots/UpdateJot.java  1
- app/src/main/java/com/larryhsiao/nyx/jots/NewJot.java  1
- app/src/main/java/com/larryhsiao/nyx/android/jot/JotContentFragment.java  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/LarryHsiao/Nyx/pullRequest?prid=5095791&bid=16761474)

**Comment by LarryHsiao** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
           

Complexity increasing per file
==============================
- app/src/main/java/com/larryhsiao/nyx/jots/NewJot.java  1
- app/src/main/java/com/larryhsiao/nyx/android/jot/JotContentFragment.java  3
- app/src/main/java/com/larryhsiao/nyx/jots/UpdateJot.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/LarryHsiao/Nyx/pullRequest?prid=5095791&bid=16761474)

**Comment by 0pdd** [comment]
@LarryHsiao the puzzle [#259](https://github.com/LarryHsiao/Nyx/issues/259) is still not solved.

**Comment by 0pdd** [comment]
@LarryHsiao the only puzzle [#259](https://github.com/LarryHsiao/Nyx/issues/259) is solved here.

**Comment by 0pdd** [comment]
@LarryHsiao the puzzle [#283](https://github.com/LarryHsiao/Nyx/issues/283) is still not solved; solved: [#259](https://github.com/LarryHsiao/Nyx/issues/259).

**Pull request merged**